WordPress/Classes/Services/CommentService.m

```diff
@@ -801,19 +803,21 @@ - (void)deleteUnownedComments
 #pragma mark - Blog centric methods
 // Generic moderation
 - (void)moderateComment:(Comment *)comment
-             withStatus:(NSString *)status
+             withStatus:(int)status
```

Comment:
minor: should we use `CommentStatusType` here instead of `int`?
```suggestion
             withStatus:(CommentStatusType)status
```

Reply:
Woops. Of course. Thank you!